src/bootstrap.js

```diff
@@ -4,7 +4,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // for now we ship a stub, that can be upgraded as-needed after release
-function startup() {}
-function shutdown() {}
-function install() {}
-function uninstall() {}
+function startup() {} // eslint-disable-line no-unused-vars
```

Comment:
I prefer not leave eslint comments to production code. Or update the function with parameters like:
```
function startup(data, reason) {
  // Do nothing in this version
}
```